package.json

```
@@ -1,7 +1,7 @@
 {
   "name": "hubot-github-deployments",
   "description": "Integrate with GitHub deployment API",
-  "version": "1.2.3",
+  "version": "1.2.4",
```

No need to bump the version here. We're using `[release-it](https://github.com/release-it/release-it)` to handle that process. 

```suggestion
  "version": "1.2.3",
```